lib/logstash/inputs/sqs.rb

```diff
@@ -137,34 +138,27 @@ def run(output_queue)
   end # def run
 
   def teardown
-    @sqs_queue = nil
+    @sqs = nil
```

Comment:
I think explicitly doing isn't needed, once the plugin isn't used the GC will clear the memory.